resources/bundled_packages/cura.json

```diff
@@ -492,6 +492,23 @@
             }
         }
     },
+    "UFPReader": {
+        "package_info": {
+            "package_id": "UFPReader",
+            "package_type": "plugin",
+            "display_name": "UFP Reader",
+            "description": "Provides support for reading Ultimaker Format Packages.",
+            "package_version": "1.0.0",
+            "sdk_version": "6.0",
```

Comment:
6.0 is not a valid semver number, it should be in the x.x.x format.

Reply:
This format is like this in the entire file. It's out of the scope of this PR to change the `sdk_version` for all the packages.

Reply:
Still, boyscout rule should result in at least this one being correct....

Reply:
53b59220f8824156c49e4f95a70ac2345dea8652 😜